tools/ci_build/github/azure-pipelines/templates/android-java-api-aar-test.yml

```diff
@@ -14,6 +14,11 @@ parameters:
   type: string
   default: 'onnxruntime-android'
 
+- name: QnnSDKVersion
+  displayName: QNN SDK version
+  type: string
+  default: '2.26.0.240828'
```

Comment:
Should we define the current QNN version in a separate template so it's not hardcoded in multiple places?

Reply:
I am also getting the QNN_VERSION from the sdk.yaml file in the qnn sdk download step. We can then use that variable wherever required